cfdtools/meshbase/_mesh.py

```diff
@@ -215,21 +211,14 @@ def face_in_nodelist(face, nodelist):
                 nodeset = set(boco.index.list())
                 # get all face index whose nodes are all in nodeset
                 listface_index = [
-                    i
-                    for i, _ in filter(
-                        lambda t: face_in_nodelist(t[1], nodeset), index_face_tuples
-                    )
+                    i for i, _ in filter(lambda t: face_in_nodelist(t[1], nodeset), index_face_tuples)
                 ]
                 boco.geodim = 'bdface'
                 boco.index = _conn.indexlist(ilist=listface_index)
                 # print(boco.name, len(nodeset), len(boco.index.list()))
 
     def list_boco_index(self):
-        return list(
-            itertools.chain(
-                *[boco.index.list() for _, boco in self._bocos.items()]
-            )
-        )
+        return list(itertools.chain(*[boco.index.list() for _, boco in self._bocos.items()]))
```

Comment:
return list(itertools.chain(*[boco.index.list() for boco in self._bocos.values()]))